qutip/tests/test_processor.py

```diff
@@ -324,11 +324,19 @@ def testDrift(self):
         Test for the drift Hamiltonian
         """
         processor = Processor(N=1)
-        processor.add_drift(sigmaz(), 0)
-        tlist = np.array([0., 1., 2.])
-        processor.add_pulse(Pulse(identity(2), 0, tlist, False))
+        processor.add_drift(sigmax() / 2, 0)
+        tlist = np.array([0., np.pi])
```

Comment:
Could we also add a test with more than two items in tlist? Such a test would catch the bug above, I think.

Reply:
Thanks!